`Cannot find module 'typescript'` when using `npx ts-node`

We have been using `npx ts-node` to run a specific script for quite some while now, and with update 10.2.0 that script started to fail.

### Expected Behavior

Script works.

### Actual Behavior

```
Cannot find module 'typescript'
Require stack:
- /home/runner/.npm/_npx/2763/lib/node_modules/ts-node/dist/index.js
- /home/runner/.npm/_npx/2763/lib/node_modules/ts-node/dist/repl.js
- /home/runner/.npm/_npx/2763/lib/node_modules/ts-node/dist/bin.js
```


### Steps to reproduce the problem

https://github.com/prisma/e2e-tests/pull/1982


### Minimal reproduction

-

<!--

This link explains why we ask for a minimal reproduction.  Thank you in advance!
https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff

You can create a reproduction here:
https://github.com/TypeStrong/ts-node-repros
-->

### Specifications

<!-- `ts-node -vv` will print ts-node, node, and TypeScript versions -->
* ts-node version: 10.2.0
* node version: 12.22.4
* TypeScript version: -
* tsconfig.json, if you're using one:
```
{
  "compilerOptions": {
    "esModuleInterop": true,
    "lib": ["ESNext", "dom"]
  }
}
```
* Operating system and version: Ubuntu 20.04.2 LTS (GH Actions)
* If Windows, are you using WSL or WSL2?:

--- a/src/index.ts
+++ b/src/index.ts
@@ -23,7 +23,6 @@ import {
   ModuleTypeClassifier,
 } from './module-type-classifier';
 import { createResolverFunctions } from './resolver-functions';
-import { ScriptTarget } from 'typescript';
 
 export { TSCommon };
 export {
@@ -551,7 +550,7 @@ export function create(rawOptions: CreateOptions = {}): Service {
   ];
 
   // Experimental REPL await is not compatible targets lower than ES2018
-  const targetSupportsTla = config.options.target! >= ScriptTarget.ES2018;
+  const targetSupportsTla = config.options.target! >= ts.ScriptTarget.ES2018;
   if (options.experimentalReplAwait === true && !targetSupportsTla) {
     throw new Error(
       'Experimental REPL await is not compatible with targets lower than ES2018'

